src/uri/include/sourcemeta/jsontoolkit/uri.h

```diff
@@ -151,9 +153,10 @@ class SOURCEMETA_JSONTOOLKIT_URI_EXPORT URI {
   /// const sourcemeta::jsontoolkit::URI
   /// uri{"https://www.sourcemeta.com/foo/bar"};
   /// assert(uri.path().has_value());
-  /// assert(uri.path().value() == "/foo/bar");
+  /// assert(uri.path().value().at(0) == "foo");
+  /// assert(uri.path().value().at(1) == "bar");
   /// ```
-  [[nodiscard]] auto path() const -> std::optional<std::string>;
+  [[nodiscard]] auto path() const -> std::optional<std::span<std::string>>;
```

Comment:
Maybe I'm totally wrong, but I *think* you can do const here:

```suggestion
  [[nodiscard]] auto path() const -> std::optional<std::span<const std::string>>;
```

Reply:
Yeah :)

Reply:
gcc is not happy with that haha!